dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -116,6 +124,97 @@ const brandExtensionLink = css`
 	}
 `;
 
+const lineStyle = css`
+	background-color: ${brand[600]};
+	content: '';
+	display: block;
+	height: 1px;
+	left: 50px;
+	width: 100%;
+	right: 0;
+	margin-left: 50px;
+`;
+
+const searchBar = css`
+	${from.desktop} {
+		display: none;
+	}
+	box-sizing: border-box;
+	display: block;
+	margin-left: 13px;
+	max-width: 23.75rem;
+	position: relative;
+	margin-bottom: 24px;
+	margin-right: 41px;
+	padding-bottom: 15px;
+`;
+
+const searchInput = css`
+	${textSans.large()}
+	background-color: rgba(255,255,255, .1);
```

Comment:
There’s a `transparentColour` helper you can import, and use like so: `transparentColour(neutral[100], 0.1)`